docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -0,0 +1,97 @@
+\page callback-functions Using Callback Functions
+
+When you create or get an object from Discord, you send the request to its API and in return you get either an error or the object you requested/created. Let's create a message and get it:
+
+~~~~~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+#include <fmt/format.h> // used to format text, install and define FMT_HEADER_ONLY to use
+
+struct GetMessage {
+    dpp::snowflake message_id = 1146725845457719367ULL;
+    dpp::snowflake channel_id = 1112789762521182211ULL;
+};
+
+int main() {
+    GetMessage gm;
+
+    std::string BOT_TOKEN = "Token Was here";
+
+    /* the second argument is a bitmask of intents - i_message_content is needed to get messages */
+    dpp::cluster bot(BOT_TOKEN, dpp::i_message_content | dpp::i_default_intents);
+
+    bot.on_log(dpp::utility::cout_logger());
+
+    bot.on_slashcommand([&bot, gm](const dpp::slashcommand_t& event) -> void {
+        if (event.command.get_command_name() == "msg-get") {
+
+            // get a message using its own and channel's ID
+            bot.message_get(gm.message_id, gm.channel_id, [&bot, event](const dpp::confirmation_callback_t& callback) -> void {
+                if (callback.is_error()) { // catching an error to log it
+                    bot.log(dpp::loglevel::ll_error, callback.get_error().message);
+                    return;
+                }
+
+                // callback.value will contain the message we are getting but the type will be std::variant, so we will have to get the message itself.
+                auto message = callback.get <dpp::message>();
+                //std::get <dpp::message>(callback.value) would give the same result
```

Comment:
Misplaced spaces, should be `// std::get<dpp::message>(callback.value) would give the same result`

Reply:
Oh, come on, how could I do that? Of course, I'm fixing that. Without irony readability is screwed.

Reply:
Fixed